src/webview/MessageList.js

```diff
@@ -123,6 +125,8 @@ const webviewAssetsUrl = new URL('webview/', assetsUrl);
 const baseUrl = new URL('index.html', webviewAssetsUrl);
 
 class MessageListInner extends React.Component<Props> {
+  static contextType = ThemeContext;
+  context: ThemeData;
   webviewRef = React.createRef<React$ElementRef<typeof WebView>>();
```

Comment:
nit: blank line here, to visually group the two context-related lines apart from these next few lines

(The next few lines are declaring/initializing local state that our own code in this class manages, whereas `.context` is a [special name known to React](https://reactjs.org/docs/context.html#classcontexttype) that the React engine manages. So one wants to think of them in separate categories.)